src/app/parcels/getExpandedParcelDetails.ts

```diff
@@ -1,19 +1,44 @@
 import { Schema } from "@/databaseUtils";
 import { Data } from "@/components/DataViewer/DataViewer";
 import supabase from "@/supabaseClient";
-import { DatabaseError } from "@/app/errorClasses";
 import { EventTableRow } from "./EventTable";
 import { logErrorReturnLogId } from "@/logger/logger";
-export type RawParcelDetails = Awaited<ReturnType<typeof getRawParcelDetails>>;
+import {
+    formatAddressFromClientDetails,
+    formatBreakdownOfChildrenFromFamilyDetails,
+    formatHouseholdFromFamilyDetails,
+} from "@/app/clients/getExpandedClientDetails";
+import { formatDateTime, formatDatetimeAsDate } from "@/common/format";
+
+type FetchExpandedParcelDetailsResult =
```

Comment:
I've been using "...Response" instead of "...Result", again should we choose one to be consistent, don't mind which

Reply:
idm which either, I think both exist in the code base now. Slight preference on Result because Response would imply it's the response for some sort of request (the data coming directly out of a supabase call would be a response, but not after we do a potential data manipulation / make multiple calls).

We can try standardise to Result as we go along.